ai-service/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+	compileOnly group: 'javax.websocket', name: 'javax.websocket-api', version: '1.1'
```

Comment:
this can be removed since we should be getting it from the JavaEE 8.0 dependency in our root build.gradle file